### ISSUE TITLE
All pages must contain a level 1 heading

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/ppa-django#235]
on all pages: [Page must contain a level-one heading](https://dequeuniversity.com/rules/axe/3.1/page-has-heading-one?application=AxeChrome) (classified as “moderate”)

Affected pages:
- home page (no h1)
- archive page (no h1)
- digitized work detail page (item title should be h1 instead of h3)

Can we configure pa11y to report on this?

## dev notes
- [x] add a hidden h1 where there is none
- [x] convert digitized work detail page heading to h1 & style

**Comment by thatbudakguy** [comment]
tested with aXe on the above pages in QA.
